dashboard/lib/sqs/queue_processor_config.rb

```diff
@@ -4,42 +4,33 @@ module SQS
 
   # Immutable configuration for a queue processor.
   class QueueProcessorConfig
-    attr_reader :queue_uri, :handler, :num_workers_per_processor, :num_processors
+    attr_reader :queue_url, :handler, :num_workers_per_processor, :num_processors, :logger
     attr_reader :initial_max_rate, :dcdo_max_rate_key
-    attr_reader :max_wait_time, :max_batch_size
 
-    # @param [String] queue_uri URI to the SQS queue.
+    # @param [String] queue_url URI to the SQS queue.
     # @param [Integer] initial_max_rate Initial max rate (in messages per second) or 0 for no limit.
     # @param [String] dcdo_max_rate_key DCDO key for the dynamic max rate.
```

Comment:
I'll change that.  Note though that the config read by the queue processor main program is JSON, so we won't be able to specify arbitrary procs in the config.